ci/cluster-api/vsphere/templates/cluster.yaml

```diff
@@ -33,13 +64,13 @@ spec:
       name: NETWORKNAME
     providerConfig:
       cloud:
-        controllerImage: gcr.io/cloud-provider-vsphere/cpi/release/manager:v1.1.0
+        controllerImage: gcr.io/cloud-provider-vsphere/cpi/release/manager:v1.0.0
```

Comment:
A bit curious: The new capv image release number is smaller than previous one?

Reply:
Yes. They are using an older one. https://github.com/kubernetes-sigs/cluster-api-provider-vsphere/blob/master/examples/default/cluster/cluster.yaml#L43